libsql_client/dbapi2/hrana.py

```diff
@@ -55,11 +56,21 @@ def _create_hrana_connection(
     session: aiohttp.ClientSession,
     url: str,
 ) -> HranaConn:
-    config = _expand_config(url, auth_token=None)
+    config = _expand_config(url, auth_token=None, tls=None)
 
-    # TODO: maybe handle this inside HranaConn?
-    if config.scheme == "libsql":
-        config = config._replace(scheme="wss")
+    if config.scheme not in ("ws", "wss"):
```

Comment:
i think these are better done inside `HranaConn`, so it will work both here and for the regular client.

Reply:
Fixed